example/screens/ChoiceCustomizationScreen.js

```diff
@@ -1,180 +1,181 @@
-            </RkChoiceGroup>
+        </TouchableOpacity>
+        <TouchableOpacity choiceTrigger>
+          <View style={styles.checkRow}>
+            <RkText rkType='bold'>Example 2</RkText>
+            <RkChoice rkType='posNegClearCheck' />
           </View>
-        </View>
-
-        <View style={[UtilStyles.section, UtilStyles.bordered]}>
-          <RkText rkType='header'>Radio components 4</RkText>
-          <View style={UtilStyles.columnContainer}>
-            <RkChoiceGroup radio selectedIndex={0}>
-              <TouchableOpacity choiceTrigger>
-                <View style={[styles.radioRow, styles.spaceBottom]}>
-                  <RkChoice rkType='radio smileColor'/>
-                  <RkText rkType='bold' style={{marginLeft: 16.5}}>Option 1</RkText>
-                </View>
-              </TouchableOpacity>
-              <TouchableOpacity choiceTrigger>
-                <View style={styles.radioRow}>
-                  <RkChoice rkType='radio smileColor'/>
-                  <RkText rkType='bold' style={{marginLeft: 16.5}}>Option 2</RkText>
-                </View>
-              </TouchableOpacity>
-              <TouchableOpacity choiceTrigger>
-                <View style={styles.radioRow}>
-                  <RkChoice rkType='radio smileColor'/>
-                  <RkText rkType='bold' style={{marginLeft: 16.5}}>Option 3</RkText>
-                </View>
-              </TouchableOpacity>
-            </RkChoiceGroup>
+        </TouchableOpacity>
+        <TouchableOpacity choiceTrigger>
+          <View style={styles.checkRow}>
+            <RkText rkType='bold'>Example 3</RkText>
+            <RkChoice
+              selected
+              rkType='clear'
+              style={{ borderWidth: 1, borderRadius: 20, borderColor: '#da7b00' }}
+              contentStyle={{ tintColor: '#ff9102' }}
+            />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity choiceTrigger>
+          <View style={styles.checkRow}>
+            <RkText rkType='bold'>Example 4</RkText>
+            <RkChoice rkType='whiteGreenCheck' selected />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity choiceTrigger>
+          <View style={styles.checkRow}>
+            <RkText rkType='bold'>Example 5</RkText>
+            <RkChoice rkType='star' selected />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity choiceTrigger>
+          <View style={styles.checkRow}>
+            <RkText rkType='bold'>Example 6</RkText>
+            <RkChoice selected rkType='rainbowCat' />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity choiceTrigger>
+          <View style={styles.checkRow}>
+            <RkText rkType='bold'>Example 7</RkText>
+            <RkChoice
+              style={{ backgroundColor: 'transparent', borderWidth: 0 }}
+              renderContentFunction={this.renderCustomChoiceContent}
+              selected
```

Comment:
Is there a chance to add a rule which will stand that value is mandatory for all attributes?